pywhat/printer.py

```diff
@@ -157,7 +157,7 @@ def print_raw(self, text: dict, text_input) -> str:
                         output_str += "\n[bold #D7Afff]Exploit: [/bold #D7Afff]" + i["Regex Pattern"]["Exploit"]
                     output_str += "\n\n"
 
-        if output_str == "":
+        if output_str == "" and self.bug_bounty_mode:
```

Comment:
I think you meant to say `not self.bug_bounty_mode` here. Try that? The tests should work :)